reference.md

```diff
@@ -18,46 +18,53 @@ _____
 - `ls -a` - list all files, including hidden files
 - `ls *.txt` - list all files that end with `.txt`
 
-**`cd`** change directory
+**`cd`** - change directory
 
-  `cd pathname` - takes you to the directory specified by `pathname`
-  
-  `cd ~` -  takes you to your home directory
+- `cd pathname` - takes you to the directory specified by `pathname`
+- `cd ~` -  takes you to your home directory
```

Comment:
The changes up to here are mostly cosmetic. Please don't merge them.

If we find it important to add `-` as a seperator, please create an issue to discuss, or a separate PR.

Reply:
Hi @katrinleinweber as part of the checklist to have this lesson published the Carpentries asked us to go through the lessons with a fine tooth comb.  An issue was created that there was missing content in the reference document.  So while doing that updating I also made sure to also make the document more consistent, which is why there are cosmetic changes as well as additions of new code. I will talk to the rest of the team about these changes at our next meeting.

Reply:
Did the need for consistency arise after you added new content in a different style than the existing commands list? It's not my decision in the end, but please consider the usefulness of the [`git blame` view](https://github.com/LibraryCarpentry/lc-shell/blame/847ac8a0ed184b8a093327ec45491976a3af83f9/reference.md). Right now, content and style changes are mashed into 1 commit, which makes researching the reasons for particular changes harder in the future.

Reply:
@katrinleinweber The inconsistencies were in the original document, they had two different styles being used.  I can understand splitting out consistency and new additions into two pull requests for lessons but I'm not sure how much it matters for a reference file that not everyone shares.  Like I said I will add this item to my agenda for the next lesson maintainers meeting agenda.